src/xrPhysics/CalculateTriangle.h

```diff
@@ -1,33 +1,33 @@
 #pragma once
 
+// Standard includes
 #include "ExtendedGeom.h"
 #include "MathUtils.h"
-
 #include "Geometry.h"
+#include "xrCDB/xr_area.h"
 #include "tri-colliderknoopc/dTriColliderMath.h"
-
 #include "ode_redefine.h"
-#include "xrCDB/xr_area.h"
```

Comment:
Don't remove logical separation between includes groups. In other words, don't remove empty lines between includes.